### PR TITLE
Improve has good jugement

### PR DESCRIPTION
**Pull request by itmilos**
Used map and map_or method instead of an if let statement.

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-each-crate-macos
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2700562

File: bin/node/runtime/src/impls.rs [hunk 1]
```diff
@@ -60,16 +60,13 @@ impl IdentityVerifier<AccountId> for AllianceIdentityVerifier {
...
-		if let Some(judgements) =
-			crate::Identity::identity(who).map(|registration| registration.judgements)
-		{
-			judgements
-				.iter()
-				.any(|(_, j)| matches!(j, Judgement::KnownGood | Judgement::Reasonable))
-		} else {
-			false
-		}
+		crate::Identity::identity(who)
```
Comment (ggwpez) [inline bin/node/runtime/src/impls.rs:63]:
```suggestion
		use pallet_identity::Judgement;
		crate::Identity::identity(who)
```

**Comment by ggwpez** [comment]
bot merge

**Comment by ggwpez** [comment]
bot fmt

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**